src/util/validationUtils.js

```diff
@@ -8,7 +8,7 @@ const { BadRequestError, NotImplementedError } = require('./errorUtils');
  */
 function validateEvalMeasureParams(query) {
   const REQUIRED_PARAMS = ['periodStart', 'periodEnd'];
-  const UNSUPPORTED_PARAMS = ['measure', 'practitioner', 'lastReceivedOn'];
+  const UNSUPPORTED_PARAMS = ['measure', 'lastReceivedOn'];
 
   checkRequiredParams(query, REQUIRED_PARAMS, '$evaluate-measure');
```

Comment:
We should add a check here like we have in the gaps validation to make sure that Practitioner is is of the form Practitioner/id, and a test that the validation utils throw the right error in that case

Reply:
[3cc81eb](https://github.com/projecttacoma/deqm-test-server/pull/96/commits/3cc81eb3436cb880f935f0bb48a6cbd6eee513c9)